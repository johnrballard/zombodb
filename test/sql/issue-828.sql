CREATE TABLE issue828
(
    id        SERIAL8                    NOT NULL PRIMARY KEY,
    full_text zdb.fulltext_with_shingles NOT NULL
);
CREATE INDEX es_idxhyphenshinglestest_var
    ON issue828 USING zombodb ((issue828.*));

insert into issue828(full_text)
values ('please remove me from your e-mail list');
insert into issue828(full_text)
values ('please send me your e-mails');

SELECT *
FROM issue828
WHERE issue828 ==> '(full_text : "e-mail*" )';

DROP TABLE issue828;