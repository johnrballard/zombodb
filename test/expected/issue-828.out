CREATE TABLE issue828
(
    id        SERIAL8                    NOT NULL PRIMARY KEY,
    full_text zdb.fulltext_with_shingles NOT NULL
);
CREATE INDEX es_idxhyphenshinglestest_var
    ON issue828 USING zombodb ((issue828.*));
insert into issue828(full_text)
values ('please remove me from your e-mail list');
insert into issue828(full_text)
values ('please send me your e-mails');
SELECT *
FROM issue828
WHERE issue828 ==> '(full_text : "e-mail*" )';
 id |               full_text                
----+----------------------------------------
  1 | please remove me from your e-mail list
  2 | please send me your e-mails
(2 rows)

DROP TABLE issue828;
